linkerd/app/outbound/src/sidecar.rs

```diff
@@ -74,15 +89,105 @@ impl Outbound<()> {
 
                     http::Routes::Endpoint(Remote(ServerAddr(*orig_dst)), Default::default())
                 };
-                let routes = match (*parent).profile.clone() {
-                    Some(profile) => {
-                        let mut rx = watch::Receiver::from(profile);
-                        let init = mk_routes(&*rx.borrow_and_update());
-                        http::spawn_routes(rx, init, move |profile: &profiles::Profile| {
-                            Some(mk_routes(profile))
+
+                let mk_policy_routes = move |policy: &policy::ClientPolicy| {
+                    // The `HttpRoutes` types are currently constructed with an
+                    // `Addr`, which is used to implement `Param<LogicalAddr>`,
+                    // required by the rest of the stack for metrics labels,
+                    // tap, et cetera.
+                    //
+                    // Unlike ServiceProfiles, OutboundPolicies do not contain a
+                    // single authority which can be used as a logical address.
+                    // Instead, the target of the policy is identified by
+                    // structured `policy::Meta` metadata. Since we don't have
+                    // an authority here, we use the original dest address as
+                    // the logical address for the time being.
+                    //
+                    // TODO(eliza): eventually, metrics and tap labels should be
+                    // generated from the policy's structured metadata, rather
+                    // than an address.
```

Comment:
```suggestion
                    // We used the `OrigDstAddr` to lookup the policy.
                    // Attach it to routes so it can use that metadata.
```

Reply:
basically, i think this is mostly a distraction from what this code is doing.